src/components/Map.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import GoogleMapReact from 'google-map-react';
+
+function Map(){
+    const defaultLocation = {
+        center:{
+            lat: 59.95,
+            lng: 30.33
+        },
+        zoom: 11
+
+    };
+    return (
+        <div style={{ height: '100vh', width: '100%' }}>
+          <GoogleMapReact
+            bootstrapURLKeys={{ key:'AIzaSyA_jF-TPUl8qTMZ3BKFTrFOolH9wR7NOz4'}}
```

Comment:
Should the key be kept hidden in another part of the code or is it not essential for this one?

Reply:
Good question! I asked @segdeha about this during our Stand Up on Sunday, and it was mentioned that the way the key is created, certain permissions are already attached to it, so the key doesn't need to be obscured.

Reply:
Yep, I think this is fine for now. I ended up with so many props I was passing in to this component that I moved them out to a separate projects and passed them in as `{...mapProps}`, but we shouldn’t optimize prematurely.